showcase/package.json

```diff
@@ -75,6 +75,7 @@
     "ember-concurrency": "^4.0.2",
     "ember-deep-tracked": "^2.0.0",
     "ember-load-initializers": "^2.1.2",
+    "ember-math-helpers": "^3.0.0",
```

Comment:
question: I've not seen this in use; am I missing something?

Reply:
Its just used for this example in the showcase: https://hds-showcase-git-hds-3965-advanced-table-clean-hashicorp.vercel.app/components/advanced-table#with-nested-rows-and-cells-that-span-rows

https://github.com/hashicorp/design-system/blob/0ecbd25af1e65f290eee90d525f8acff0228f65c/showcase/app/templates/components/advanced-table.hbs#L126

Reply:
right, I missed that instance – thanks!